src/main/webapp/WEB-INF/tags/admin/email/compose.tag

```diff
@@ -66,7 +69,7 @@
             </span>
         </form>
 
-        <div id="documentBaseUrl">${emailToEdit.appUrl}</div>
+        <div id="documentBaseUrl">${APP_URL}</div>
```

Comment:
Minor issue: I believe `<%= Config.APP_URL %>` will do just fine here, no need to set variable for it.